src/entry/logEntry.store.ts

```diff
@@ -0,0 +1,45 @@
+import { UUID } from "node:crypto";
+import { DatabaseService } from "../db/dbConfig";
+import { LogEntry, LogEntryError } from "./logEntry";
+
+export class LogEntryStore {
+  static instance: LogEntryStore | undefined;
+
+  static getInstance(): LogEntryStore {
+    if (LogEntryStore.instance === undefined) {
+      LogEntryStore.instance = new LogEntryStore();
+    }
+
+    return LogEntryStore.instance;
+  }
+
+  async get(
+    sessionID: UUID,
+    files: string[],
+    from: number,
+    count: number,
+  ): Promise<LogEntry[] | LogEntryError> {
+    try {
+      const query = {
+        text: `
+            SELECT *
+            FROM loggaroo.log_entry
+              WHERE session_id = $1
+                AND file_name IN $2
```

Comment:
this does not seem to be working...
you need to use 

```suggestion
                AND file_name = ANY($2)
```